src/misc/lv_anim.h

```diff
@@ -112,44 +112,45 @@ typedef int32_t (*lv_anim_get_value_cb_t)(lv_anim_t *);
 /** Callback used when the animation is deleted*/
 typedef void (*lv_anim_deleted_cb_t)(lv_anim_t *);
 
+/**< Parameter used when path is custom_bezier */
 typedef struct {
     int16_t x1;
     int16_t y1;
     int16_t x2;
     int16_t y2;
-} lv_anim_bezier3_para_t; /**< Parameter used when path is custom_bezier*/
+} lv_anim_bezier3_para_t;
 
 /** Describes an animation*/
 struct lv_anim_t {
-    void * var;                                 /**<Variable to animate*/
-    lv_anim_exec_xcb_t exec_cb;                 /**< Function to execute to animate*/
-    lv_anim_custom_exec_cb_t custom_exec_cb;/**< Function to execute to animate,
-                                                 same purpose as exec_cb but different parameters*/
-    lv_anim_start_cb_t start_cb;         /**< Call it when the animation is starts (considering `delay`)*/
-    lv_anim_completed_cb_t completed_cb; /**< Call it when the animation is fully completed*/
-    lv_anim_deleted_cb_t deleted_cb;     /**< Call it when the animation is deleted*/
-    lv_anim_get_value_cb_t get_value_cb; /**< Get the current value in relative mode*/
-    void * user_data;                    /**< Custom user data*/
-    lv_anim_path_cb_t path_cb;         /**< Describe the path (curve) of animations*/
-    int32_t start_value;               /**< Start value*/
-    int32_t current_value;             /**< Current value*/
-    int32_t end_value;                 /**< End value*/
-    int32_t duration;                /**< Animation time in ms*/
-    int32_t act_time;            /**< Current time in animation. Set to negative to make delay.*/
-    uint32_t playback_delay;     /**< Wait before play back*/
-    uint32_t playback_duration;      /**< Duration of playback animation*/
-    uint32_t repeat_delay;       /**< Wait before repeat*/
-    uint32_t repeat_cnt;         /**< Repeat count for the animation*/
+    void * var;                               /**<Variable to animate*/
+    lv_anim_exec_xcb_t exec_cb;               /**< Function to execute to animate*/
+    lv_anim_custom_exec_cb_t custom_exec_cb;  /**< Function to execute to animate,
+                                               * same purpose as exec_cb but different parameters*/
```

Comment:
Could we match all comments to add a space after and before `*`? It should be easier to use regexpression.